internal/stacks/stack_manager.go

```diff
@@ -505,15 +509,23 @@ func (s *StackManager) createMember(id string, index int, options *types.InitOpt
 		ExposedFireflyAdminSPIPort: serviceBase + 1, // note shared blockchain node is on zero
 		ExposedConnectorPort:       serviceBase + 2,
 		ExposedUIPort:              serviceBase + 3,
-		ExposedDatabasePort:        serviceBase + 4,
-		ExposedDataexchangePort:    serviceBase + 5,
-		ExposedIPFSApiPort:         serviceBase + 6,
-		ExposedIPFSGWPort:          serviceBase + 7,
 		External:                   external,
 		OrgName:                    options.OrgNames[index],
 		NodeName:                   options.NodeNames[index],
 	}
-	nextPort := serviceBase + 8
+
+	nextPort := serviceBase + 4
+	if options.MultipartyEnabled {
+		member.ExposedDatabasePort = serviceBase + nextPort
```

Comment:
The database will exist for multiparty and non-multiparty.

Reply:
Thanks good catch